action.yml

```diff
@@ -28,13 +28,28 @@ inputs:
     description: 'The architecture of Flutter SDK executable (x64 or arm64)'
     required: false
     default: ${{ runner.arch }}
+  false-positive-cache:
```

Comment:
I think we can use something like:

```
cache-key-suffix-hash: true
```

Then the script will add a manifest hash to the end of the cache key, like `-da39a3e`.

What do you think @KKimj?

Reply:
Yep. It looks better than `false-positive-cache`.
Starting with `cache` is better.

But, I wonder, as it's not as easy to know by `cache-key-suffix-hash`, name that this option exists to ensure the latest Flutter SDK.

Reply:
@subosito 
Anyway, What do you think about `cache-key with manifest-hash`?

Reply:
@KKimj Yeah, I think reusing `cache-key` is better. Let's say, we support special syntax like: `:manifest-hash:`, it will then auto replaced by the actual manifest hash.

We also need to support combined with a plain string like `flutter-:manifest-hash:` which then transformed into `flutter-da39a3e`.

I think we can also use the shorter forms, like `:ref:` (as per https://docs.flutter.dev/development/tools/sdk/releases) or `:hash:` (as per JSON manifest).

Reply:
@subosito 
Hello, LTNS!

How about forcing cache with `flutter-ref`, without input?

Reply:
@subosito 
Hello~ LTNS
Give some comments,
And I will condunct some codes!

Reply:
Hi @KKimj, I have implemented something different for this issue in #175. Please take a look at it and let me know your thoughts. Thanks.

Reply:
@subosito 
I close this PR
#175 is LGTM for me!
Thanks.